pkg/plugin/errors.go

```diff
@@ -14,10 +14,39 @@
 
 package plugin
 
-import "github.com/conduitio/conduit/pkg/foundation/cerrors"
+import (
+	"fmt"
+
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+)
 
 var (
 	ErrStreamNotOpen    = cerrors.New("stream not open")
 	ErrPluginRunning    = cerrors.New("plugin is running")
 	ErrPluginNotRunning = cerrors.New("plugin is not running")
 )
+
+type ValidationError struct {
+	Err error
```

Comment:
Nitpick: I wouldn't expose a field unless there's a reason, and here I don't see one. The underlying error can be accessed by `Unwrap` already.

Reply:
I needed to expose it because I needed to assign the error when creating it in the "Validate" method.

> if err != nil {
> 			return &plugin.ValidationError{Err: err}
> 		}

Reply:
If we manage to pull the logic into the plugin service this shouldn't be an issue anymore.